### PR TITLE
Spreadsheet style validation error tables

### PR DESCRIPTION
**Pull request by Bjwebb**
#1131 

[Screenshots and demo links.](https://github.com/OpenDataServices/cove/issues/1131#issuecomment-471427123)

In order to provide context cells for the tables, I'm re-opening the xlsx file from within CoVE (previously just within flatten-tool).

All of these changes are specific to 360Giving CoVE only. As part of this I've made a copy of `validation_table.html`.

### CODE REVIEW

File: cove_360/lib/threesixtygiving.py [hunk 1]
```diff
@@ -104,31 +106,115 @@ def get_grants_aggregates(json_data):
...
+        prev_number = next(prev_numbers, -1)
+        next_number = next(next_numbers, None)
+        if prev_number != number - 1 and prev_number + 1 != number - 1 and number - 1 > 0:
+            yield number - 1
+        yield number
+        if next_number != number + 1:
+            yield number + 1
+
+
+def spreadsheet_style_errors_table(examples, openpyxl_workbook):
```
Comment (BibianaC) [inline cove_360/lib/threesixtygiving.py]: Is it possible to add a test?

**Comment by BibianaC** [comment]
@Bjwebb I guess it is fine what you wrote but in this case I would feel more comfortable if someone else reviews it too.

File: cove_360/lib/threesixtygiving.py (hunk 1 above)
Comment (Bjwebb) [reply cove_360/lib/threesixtygiving.py]: Good idea, I've done so.

**Comment by Bjwebb** [comment]
@BibianaC Thanks. I got some feedback from Rob on a call, so I'll be working through that, and then I'll assign it to someone else to review the code.

**Pull request merged**